doxygen/examples/rtr_mgr.c

```diff
@@ -73,14 +106,38 @@ int main(){
     rtr_mgr_start(conf);
 
     //wait till at least one rtr_mgr_group is fully synchronized with the server
-    while(!rtr_mgr_conf_in_sync(conf))
+    while(!rtr_mgr_conf_in_sync(conf)) {
+	if (connection_status)
```

Comment:
this will likely fail immediately, because connection_status is init with `-1`, or not?